### PR TITLE
*/*: -fno-common 

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: app-text/bibutils/bibutils-6.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+S="${WORKDIR}/${MY_P}"
+
+src_configure() {
+	./configure \
+		--dynamic \
+		--install-dir "${D}/usr/bin" \
+		--install-lib "${D}/usr/$(get_libdir)" || die
```
Comment (SoapGentoo) [inline app-text/bibutils/bibutils-6.10.ebuild:23]: don't want to use `${ED}`?

File: dev-lang/snobol/snobol-1.4.1-r1.ebuild [hunk 2]
```diff
@@ -14,39 +15,39 @@ SLOT="0"
...
-		-i Makefile2.m4 || die 'strip sed failed'
-	echo "ADD_OPT([${CFLAGS}])" >>${S}/local-config
-	echo "ADD_CPPFLAGS([-DUSE_STDARG_H])" >>${S}/local-config
-	echo "ADD_CPPFLAGS([-DHAVE_STDARG_H])" >>${S}/local-config
+	default
+	sed -i -e '/autoconf/s:autoconf:./autoconf:g' configure || die
+	sed -i -e 's/$(INSTALL) -s/$(INSTALL)/' Makefile2.m4 || die
+	echo "ADD_OPT([${CFLAGS}])" >> "${S}"/local-config || die
+	echo "ADD_CPPFLAGS([-DUSE_STDARG_H])" >> "${S}"/local-config || die
+	echo "ADD_CPPFLAGS([-DHAVE_STDARG_H])" >> "${S}"/local-config || die
```
Comment (SoapGentoo) [inline dev-lang/snobol/snobol-1.4.1-r1.ebuild:32]: unnecessary `"${S}"/`?

File: games-strategy/naev/Manifest [hunk 3]
```diff
@@ -1,4 +1,3 @@
...
 DIST naev-0.7.0.tar.bz2 1076658 BLAKE2B 582bbfbc16e33d16600c19928ed6810dfe5fd3165930e6a51ec9b65d25305cd0bbfaf6f4cced12c5b7f76163ed4dc36b7108e81f4a4e82dc626c6045d3fe769f SHA512 9f8a2912e9f4f29ed6e276b008d4010da3ccdd309de5bb4360c393f2704aa7aff6e18db9e5d3657b4229e911ddec9b2a3312cf182c3c48a91a007a91138d84ae
-DIST ndata-0.6.1.zip 303984255 BLAKE2B 154d8b326e2596cfe16fc646e7d482c07d7015253b6933889d12b93a2ee7e28649b1f3ffcb0c0ed2bfe9d1e3654623cab00113486d05fe6fd1262ce989bf9c25 SHA512 71c3024e1713b094b63cb0de3c70239d61c54142b715e61d5206c5eb3cb8c7a851fc1b388ecf891110dd8bb5c2b3759af71d20f39cb6e629c71b27386a57d7fe
+DIST naev-0.8.1-source.tar.gz 359781119 BLAKE2B 021824b0af031af8ff2aa565989c3d7811fcbcdc082649f208e8318b0d69b1c3c27492936a6afdc38de9448aa0dcca2f8ef73879155217e34968f81246017aa6 SHA512 8f9aede300d06b0beae12d4809cc58ac740bc1934c2fa5fdb04452cd2eb63de697e0bbf2aacf5af4878b62eee30878ed50388ee8386fbb19f9d56cb7cad8d7f2
```
Comment (SoapGentoo) [inline games-strategy/naev/Manifest:3]: thank you for bumping this

File: net-print/gtklp/gtklp-1.3.1.ebuild [hunk 4]
```diff
@@ -15,14 +15,13 @@ SLOT="0"
...
-RDEPEND="x11-libs/gtk+:2
+RDEPEND="
 	net-print/cups
+	x11-libs/gtk+:2
 	nls? ( sys-devel/gettext )
 	ssl? ( dev-libs/openssl:0= )"
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
-
-DOCS="AUTHORS BUGS ChangeLog README TODO USAGE"
```
Comment (SoapGentoo) [inline net-print/gtklp/gtklp-1.3.1.ebuild:25]: are you losing `USAGE` with this? consider putting `dodoc USAGE` after the default in `src_install` if thats the case

File: sci-biology/finchtv/finchtv-1.3.1-r3.ebuild [hunk 5]
```diff
@@ -25,12 +24,10 @@ src_install() {
 	exeinto /opt/bin
 	doexe finchtv
 
-	dodoc ReleaseNotes.txt
+	dodoc -r SampleData
+	docompress -x /usr/share/doc/${PF}/SampleData
```
Comment (SoapGentoo) [inline sci-biology/finchtv/finchtv-1.3.1-r3.ebuild:28]: personally, I would install `examples` instead, given that's a semi-standardised location

File: sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch [hunk 6]
```diff
@@ -0,0 +1,13 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -457,8 +457,8 @@ AH_TEMPLATE(DRBD_LEGACY_84, [Include support for drbd-8.4 kernel code])
+ AH_TEMPLATE(WINDRBD, [Include support for windrbd (do not use this under Linux)])
+ 
+ AC_SUBST(DRBD_LIB_DIR, [$localstatedir/lib/$PACKAGE_TARNAME])
+-AC_SUBST(DRBD_RUN_DIR, [$localstatedir/run/$PACKAGE_TARNAME])
+-AC_SUBST(DRBD_LOCK_DIR, [$localstatedir/lock])
++AC_SUBST(DRBD_RUN_DIR, [/run/$PACKAGE_TARNAME])
++AC_SUBST(DRBD_LOCK_DIR, [/run/lock])
```
Comment (SoapGentoo) [inline sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch:10]: it's fine, just remember this destroys Prefix support

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-text/bibutils/bibutils-6.10.ebuild (hunk 1 above)
Comment (jsmolic) [reply app-text/bibutils/bibutils-6.10.ebuild:23]: Yes, yes, I do :smile:

File: dev-lang/snobol/snobol-1.4.1-r1.ebuild (hunk 2 above)
Comment (jsmolic) [reply dev-lang/snobol/snobol-1.4.1-r1.ebuild:32]: Good catch, I missed it

File: app-text/bibutils/bibutils-6.10.ebuild (hunk 1 above)
Comment (jsmolic) [reply app-text/bibutils/bibutils-6.10.ebuild]: Done

File: dev-lang/snobol/snobol-1.4.1-r1.ebuild (hunk 2 above)
Comment (jsmolic) [reply dev-lang/snobol/snobol-1.4.1-r1.ebuild]: Done

File: net-print/gtklp/gtklp-1.3.1.ebuild (hunk 4 above)
Comment (jsmolic) [reply net-print/gtklp/gtklp-1.3.1.ebuild:25]: Done

File: sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch (hunk 6 above)
Comment (jsmolic) [reply sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch:10]: Okay. Hmm, maybe I can edit this with `sed`?

File: games-strategy/naev/Manifest (hunk 3 above)
Comment (jsmolic) [reply games-strategy/naev/Manifest:3]: np, the test phase in only failing now

File: sci-biology/finchtv/finchtv-1.3.1-r3.ebuild (hunk 5 above)
Comment (jsmolic) [reply sci-biology/finchtv/finchtv-1.3.1-r3.ebuild]: Done

**Pull request closed without merging**